components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -135,6 +137,13 @@ void RRTMGPRadiation::set_grids(const std::shared_ptr<const GridsManager> grids_
   add_field<Computed>("cldmed"        , scalar2d_layout, nondim, grid_name, "RESTART");
   add_field<Computed>("cldhgh"        , scalar2d_layout, nondim, grid_name, "RESTART");
   add_field<Computed>("cldtot"        , scalar2d_layout, nondim, grid_name, "RESTART");
+  // Cloud-top diagnostics following AeroCOM recommendation
+  add_field<Computed>("aerocom_ttop"  , scalar2d_layout, nondim, grid_name, "RESTART");
```

Comment:
Are these really needed in the restart file? Their calculation below seems to be indep of their previous value...

Reply:
Uhm, I forgot about that. This is b/c rad can be supercycled, right? I guess you can do that, but we really need to fix our I/O classes to avoid this stuff.